backend/app/core/validators/pii_remover.py

```
@@ -0,0 +1,74 @@
+                fix_value=anonymized_text
+            )
+        return PassResult(value=text)        
+
+    def run_english_presidio(self, text: str):
+        results = self.analyzer.analyze(text=text,
+                                language="en")
+        anonymized = self.anonymizer.anonymize(text=text, analyzer_results=results)
+        return anonymized.text
+
+    def run_hinglish_presidio(self, text: str):
+        return text
```

Ruff (0.14.8)

No newline at end of file

Add trailing newline

(W292)